students/12th/hyeongwoolee/js/main.js

```diff
@@ -0,0 +1,46 @@
+//게시 버튼 색깔 변경
+const commentBtn = document.getElementById("comment__add__button");
+const commentInput = document.getElementById("comment__body");
+const commentBody = document.getElementById("comment__message__content"); //댓글이 붙는 태그
+//댓글내용
+
+const addBtn = function (e) {
```

Comment:
```suggestion
const addBtn = e => {
```
arrow function에 맞게 써주세요!